src/pages/Equalizer/HeroSection/HeroSection.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import logo from '../../../assets/equalizer-img/logo.svg';
+import mobileBgPattern from '../../../assets/equalizer-img/bg-main-mobile.png';
+import tabletBgPattern from '../../../assets/equalizer-img/bg-main-tablet.png';
+import desktopPattern from '../../../assets/equalizer-img/bg-main-tablet.png';
+import bgPattern1 from '../../../assets/equalizer-img/bg-pattern-1.svg';
+import './HeroSection.css';
+
+function HeroSection() {
+  return (
+    <div>
+      <div>
+        <img
+          id="logo"
+          className="mt-10 ml-5 md:mt-20 md:ml-10 xl:ml-[100px]"
+          src={logo}
+          alt=""
+        />
+        <img
+          className=" absolute top-0 z-[-1] right-0 md:hidden  "
```

Comment:
Fix spacings in class names list in all files.
`" some-class"`
`"some-class "`
double spaces
etc.